openrtb_ext/bidders_test.go

```diff
@@ -61,3 +61,8 @@ func TestBidderListDoesNotDefineGeneral(t *testing.T) {
 	bidders := BidderList()
 	assert.NotContains(t, bidders, BidderNameGeneral)
 }
+
+func TestBidderListDoesNotDefineContext(t *testing.T) {
```

Comment:
If the `BidderNameContext` is only being used in this test, can we just define that as a constant here in this specific test file? The reason being it can be otherwise easily confused with `FirstPartyDataContextExtKey` which is what we're actually using when looking at the ext to separate it out from the bidders

Reply:
I included `BidderNameContext` alongside the other bidder name definitions to act as documentation of expected cases while also providing a `BidderName` instance (`FirstPartyDataContextExtKey` is a string). I used the same pattern for `BidderNameGeneral` for the same reason.

The variables also refer to slightly different things:
`BidderNameContext` = A reserved bidder name due to a conflict with another imp.ext use case. Will eventually go away when we move entirely to imp.ext.prebid.bidder.xxx

`FirstPartyDataContextExtKey` = Name of the first party data context imp.ext field. Likely to be used more in the future when building out the first party data feature.